src/content/docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-components.mdx

```diff
@@ -0,0 +1,129 @@
+
+```
+Trying to connect to kubelet through API proxy
+```
+
+## Control plane component [#nrk8s-controlplane]
+
+New Relic deploys `nrk8s-controlplane` as a DaemonSet with `hostNetwork: true`. The configuration is structured to support autodiscover and static endpoints.
+A wide range of known defaults are provided as configuration entries, which allows you to tweak autodiscovery to your needs.
+
+You can have multiple endpoints per selector and add a probe mechanism, which automatically detects the correct one. You can use the same selector to try different configurations such as ports or protocols.
+
+Scraping configuration for the `etcd CP` component looks like the following where the same structure and features applies for all components:
+
+```yaml
+config:
+  etcd:
+    enabled: true
+    autodiscover:
+      - selector: "tier=control-plane,component=etcd"
+        namespace: kube-system
+        matchNode: true
+        endpoints:
+          - url: https://localhost:4001
+            insecureSkipVerify: true
+            auth:
+              type: bearer
+          - url: http://localhost:2381
+    staticEndpoint:
+      url: https://url:port
+      insecureSkipVerify: true
+      auth: {}
+```
+
+If `staticEndpoint` is set, the component will try to scrape it. If it can't hit the endpoint, the integration will fail so there are no silent errors when manual endpoints are configured.
+
+If `staticEndpoint` is not set, the component will iterate over the autodiscover entries looking for the first pod that matches the `selector` in the specified `namespace`, and optionally is running in the same node of the DaemonSet (if `matchNode` is set to `true`). When a pod is discovered, the component probes, issuing an http `HEAD` request, the endpoints are listed in order and scrapes the first successful probed one using the authorization type selected.
+
+The scraping logic is the same for the `etcd` component and other components.
+
+Check [control plane monitoring](/docs/kubernetes-pixie/kubernetes-integration/advanced-configuration/configure-control-plane-monitoring) to know how to configure control plane monitoring.
+
+## Helm Charts [#helm-charts]
+
+Helm is the primary recommendation to deploy New Relic with your clusters.Some of the features og Helm chart are these:
+
+* Full control of the `securityContext` for all pods
+* Full control of pod `labels` and `annotations` for all pods
+* Ability to add extra environment variables, `volumes`, and `volumeMounts`
+* Full control on the integration configuration, including which endpoints are reached, autodiscovery behavior, and scraping intervals
+* Better alignment with Helm idioms and standards
+
+Check [`README.md`](https://github.com/newrelic/nri-kubernetes/blob/main/charts/newrelic-infrastructure/README.md) to get more details on all the switches that can be flipped in the Helm Charts.
```

Comment:
Helm charts are an installation method rather than a component of the integration. I'm not sure if they fit in this page.

Reply:
I've removed it because @csongnr agrees with your comment.